Shopify/shopify-cli-extensions
# fix: ActionSet is no longer hidden when using keyboard nav

issue: #177 

## Testing

1.  Launch dev-console-app
2. Try tabbing through each component
3. It should look like this:

https://user-images.githubusercontent.com/90475219/141528903-7c13262b-6bd3-4e34-8a7f-b3005623a7ea.mov
